turbo.json

```diff
@@ -25,6 +25,10 @@
             "outputs": ["dist/**"],
             "dependsOn": ["@elizaos/plugin-node#build"]
         },
+        "@elizaos/plugin-evm#build": {
+            "outputs": ["dist/**"],
```

Comment:
Added a new build step for the plugin-evm, ensuring it has the necessary dependencies for successful builds.